src/vs/workbench/contrib/debug/browser/media/debug.contribution.css

```diff
@@ -36,7 +36,8 @@
 .codicon-debug-breakpoint-conditional.codicon-debug-stackframe::after,
```

Comment:
Here the `.codicon-debug-breakpoint-conditional.codicon-debug-stackfram` class names are kind of misused to detect a cerain type of breakpoint.
Would be better if the debug view uses a different class name to mark breakpoint

Reply:
This is actually used to detect a case where both a breakpoint exists and is on a current stackframe. Both of these have decorations of their own. Though technically we could handle that in code and make a 3rd type of decoration.